From 06e2acc756b7c209ffc0bb4151712a972c493fe0 Mon Sep 17 00:00:00 2001
From: James Page <james.page@canonical.com>
Date: Wed, 21 Aug 2024 11:39:38 +0100
Subject: [PATCH] Move dateutil types to test requirements

types-python-dateutils is only needed when running mypy static type
checking; move from runtime to test requirements.
---
 pyproject.toml                      | 1 -
 requirements/requirements-tests.txt | 1 +
 requirements/requirements.txt       | 1 -
 3 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 5ccc497a..89609bd1 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,6 @@ classifiers = [
 ]
 dependencies = [
     "python-dateutil>=2.7.0",
-    "types-python-dateutil>=2.8.10",
 ]
 requires-python = ">=3.8"
 description = "Better dates & times for Python"
diff --git a/requirements/requirements-tests.txt b/requirements/requirements-tests.txt
index 77005e0b..1f2a2ec4 100644
--- a/requirements/requirements-tests.txt
+++ b/requirements/requirements-tests.txt
@@ -6,3 +6,4 @@ pytest-cov
 pytest-mock
 pytz==2021.1
 simplejson==3.*
+types-python-dateutil>=2.8.10
diff --git a/requirements/requirements.txt b/requirements/requirements.txt
index 7d97ff8e..65134a19 100644
--- a/requirements/requirements.txt
+++ b/requirements/requirements.txt
@@ -1,2 +1 @@
 python-dateutil>=2.7.0
-types-python-dateutil>=2.8.10
